src/audio/google/google_rtc_audio_processing.c

```diff
@@ -45,10 +45,23 @@ LOG_MODULE_REGISTER(google_rtc_audio_processing, CONFIG_SOF_LOG_LEVEL);
 DECLARE_SOF_RT_UUID("google-rtc-audio-processing", google_rtc_audio_processing_uuid,
 					0xb780a0a6, 0x269f, 0x466f, 0xb4, 0x77, 0x23, 0xdf, 0xa0,
 					0x5a, 0xf7, 0x58);
+
 DECLARE_TR_CTX(google_rtc_audio_processing_tr, SOF_UUID(google_rtc_audio_processing_uuid),
 			   LOG_LEVEL_INFO);
 
+#if CONFIG_IPC_MAJOR_4
+struct sof_ipc4_aec_config {
```

Comment:
this seems like generic info no? Is there not a generic struct to use here?

Reply:
this is a generic info for all AEC modules for SOF IPC4 version

Reply:
Then it kind of begs the question whether this should be in some more generic header...? Could be that we don't really have a good place for this sort of shared headers that are used by a subset of modules, but worth a check.

Reply:
@RanderWang ping

Reply:
@cujomalainey sure, I get it. I will work on it

Reply:
done! please check it. Thanks!